src/undead/regexp.d

```diff
@@ -594,7 +594,7 @@ RegExp search(string s, string pattern, string attributes = null)
 {
     auto r = new RegExp(pattern, attributes);
     if (!r.test(s))
-    {   delete r;
+    {   r.destroy();
         assert(r is null);
```

Comment:
Fails here. You could use `__delete`.

Reply:
It's fine either way, because it's part of the documented contact.